tensorflow/lite/micro/tools/benchmarking/generic_model_benchmark.cc

```diff
@@ -45,15 +58,17 @@ namespace {
 
 using Profiler = ::tflite::MicroProfiler;
 
-constexpr int kTfLiteAbort = -9;
-
 // Seed used for the random input. Input data shouldn't affect invocation timing
 // so randomness isn't really needed.
 constexpr uint32_t kRandomSeed = 0xFB;
 
+#if !defined(MODEL_HEADER_PATH)
```

Comment:
Can we add a define like `USING_BUILTIN_MODEL` for these conditionals so that they're slightly more understandable at the usage site?

Reply:
done